docs/README.md

```diff
@@ -0,0 +1,44 @@
+# Why are we building OLM v1?
+
+Operator-lifecycle-manager's mission has been to manage the lifecycle of cluster extensions centrally and declaratively on Kubernetes clusters. Its purpose has always been to make installing, 
+running, and updating functional extensions to the cluster easy, safe, and reproducible for cluster administrators and PaaS administrators, throughout the lifecycle of the underlying cluster. 
+
+OLM so far has focused on providing unique support for these specific needs for a particular type of cluster extension, which have been coined as [operators](https://operatorhub.io/what-is-an-operator#:~:text=is%20an%20Operator-,What%20is%20an%20Operator%20after%20all%3F,or%20automation%20software%20like%20Ansible.). 
+Operators were classified as one or more Kubernetes controllers, shipping with one or more API extensions (CustomResourceDefinitions) to provide additional functionality to the cluster. 
```

Comment:
Are "Operators" classified differently now? This could just be me nitpicking but as I am reading this, the word "were" insinuates that this is no longer what an "Operator" is classified as and I didn't notice a spot where this was clarified as to what Operators are now classified as (I could have missed it).

Reply:
Good point, should be `are`.